releases/release_notes_09_2022.md

```diff
@@ -0,0 +1,273 @@
+# About 
+Welcome :-)
+
+We are thrilled to share with you Confidential Containers (COCO) release X.X.X .
+
+In this document we will take you through the content of this release, installation instructions, deploying workloads and troubleshooting if things go wrong. 
+
+# Release notes 
+
+## Goals 
+
+This release focused on the following:
+
+- **Simplicity** - Using the operator to deploy and configure
+- **Stability** - Supporting CI for the key workflows of the release
+- **Documentation** - Details instruction of how to deploy and use this release
```

Comment:
Surely one of our main focuses is also providing some kind of functionality.

Reply:
I think we need to separate the goals vs the use cases since the use cases are exactly functional.
If you can define a short point on the key functionality we provided then please do add it :-)